docs/howto/set_database_schema.rst

```diff
@@ -0,0 +1,25 @@
+Set the database schema
+-----------------------
+
+Any PostgreSQL database contains one or more *schemas*. Schemas are namespaces really,
+not much more than that. See the `PostgreSQL Schema documentation`_ for more detail on
+schemas.
+
+By default Procrastinate uses the ``public`` schema, which is PostgreSQL's default
+schema, that is the schema that every new database contains.
+
+To make Procrastinate use a specific schema, use the ``-c search_path=`` option at
+``PostgresConnector`` creation time::
+
+    app = procrastinate.App(
+        connector=procrastinate.PostgresConnector(
+            host="localhost",
+            options="-c search_path=myschema"
+        )
+    )
+
+With this the ``procrastinate schema --apply`` command will create the Procrastinate
+database objects (tables, functions, etc.) into the ``myschema`` schema. And all the
+SQL queries issued by Procrastinate will apply to the database objects of that schema.
```

Comment:
> If the connector receives an existing Pool, there is currently no way to have Procrastinate use a different schema from the rest of your application.

Reply:
I don't want to mention pools at this point, as we don't have pools in Procrastinate for the moment.